api/server.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+
+const server = express();
+
+const games = require('../arcade/arcade')
+
+server.use(express.json());
+
+
```

Comment:
Your endpoints are thorough Stefan 👍 It's good to see you using past patterns/techniques like database helpers in your code.